docs/CreateBinder.md

```diff
@@ -62,4 +62,4 @@ into this:
 
 ```[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/dotnet/try/master?urlpath=lab)```
```

Comment:
this needs to point to interactive, not to try

Reply:
Fixed